### DISCUSSION TITLE
Import Settings Broken?

### DISCUSSION THREAD

**Discussion by magnaphantasia** [mbnuqw/sidebery#1095] [General]
Tried on Firefox/Nightly/LibreWolf/Waterfox. None of these browsers let me import settings any more on the beta branch unfortunately. Not that opening all the tabs again in the fork of choice is a huge deal, but just inconvenient. 

5.0.0b33

**Comment by mbnuqw** [comment]
Issue to track: https://github.com/mbnuqw/sidebery/issues/1092
